packages-ts/gauntlet-terra-contracts/codeIds/mainnet.json

```diff
@@ -0,0 +1,7 @@
+{
+  "access_controller": 2690,
+  "ocr2": 3024,
+  "proxy_ocr2": 3202,
+  "cw3_flex_multisig": 3192,
+  "cw4_group": 3193
```

Comment:
Do we need the code id for cw20 here?

Reply:
cw20 is constant in every network

Reply:
I think it should still be explicitly referenced with the rest of the code IDs.